omega/constants.py

```diff
@@ -10,4 +10,5 @@
 SIMILARITY_THRESHOLD = 0.95
 VIDEO_DOWNLOAD_TIMEOUT = 10
 MIN_SCORE = 0.005
-FAKE_VIDEO_PUNISHMENT = -5.0
\ No newline at end of file
+FAKE_VIDEO_PUNISHMENT = -5.0
+RANDOM_DESCRIPTION_PENALTY = 0.3
```

Comment:
nit: should always add a newline at the end of files: [see here](https://www.perplexity.ai/search/is-it-wise-D1qBh8bYRbCcuKba3nTx5Q)